src/pages/playlists/[slug].tsx

```diff
@@ -14,9 +13,9 @@ type PlaylistProps = {
 const Playlist: FunctionComponent<PlaylistProps> = ({
   playlist: initialPlaylist,
 }) => {
-  const {data} = useSWR(initialPlaylist.url, fetcher)
+  const {data} = useSWR(`${initialPlaylist.slug}`, loadAuthedPlaylistForUser)
```

Comment:
is this in the code path still?

Reply:
If the playlist isnt indexed then users will be able to access `/playlists/:slug`